mifosng-provider/src/main/java/org/mifosplatform/organisation/staff/service/StaffReadPlatformServiceImpl.java

```diff
@@ -239,4 +245,38 @@ private String getStaffCriteria(final String sqlSearch, final Long officeId, fin
         sql = sql + " order by s.lastname";
         return this.jdbcTemplate.query(sql, this.staffInOfficeHierarchyMapper, new Object[] { officeId });
     }
+    
+    @Override
+	public Object[] hasAssociatedItems(final Long staffId){
+        ArrayList<String> params = new ArrayList<String>();        
+                
+        String sql =  "select c.display_name as c, g.display_name as g,l.loan_officer_id as l, s.field_officer_id as s, min(f.id)"+
```

Comment:
Why min (f.id) required in query? If it is used only for grouping then you can use group by clause. And also give some meaningful names for aliases (ex: give staff instead f).